.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+workflows:
+  version: 2
+  main:
+    jobs:
+      - node9:
+          filters:
+            branches:
+              only:
+                - master
+                - release-2.5
+                - release-2.6
+                - release-2.7
+                - circleci
+      - node8:
+          filters:
+            branches:
+              only:
+                - master
+                - release-2.5
+                - release-2.6
+                - release-2.7
+                - circleci
+      - node6:
+          filters:
+            branches:
+              only:
+                - master
+                - release-2.5
+                - release-2.6
+                - release-2.7
+                - circleci
+
+base: &base
+  environment:
+    - workerCount: 3
```

Comment:
Also, this number is tailored for maximal speed on travis. Set this to the maximum number of concurrent threads circleci will allow and actually benefit from the parallelism of (we had to measure travis with a bunch of different numbers before we landed on 3 - on a local machine we default to the number of logical processors available, which is usually optimal).

Reply:
Makes sense. I'll take a look later today.